src/main/java/org/fundacionjala/pivotal/util/CommonActions.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.pivotal.util;
+
+import org.fundacionjala.pivotal.model.pageobjects.AbstractPage;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class CommonActions extends AbstractPage {
+    private CommonActions() {
+    }
+    private void waitClick(WebElement element) {
+        wait.until(ExpectedConditions.elementToBeClickable(element));
+        element.click();
+    }
+    public static void waitAndClick(WebElement webElement) {
+        CommonActions commonActions = new CommonActions();
+        commonActions.waitClick(webElement);
+    }
+    private void waitSet(WebElement element, final String textToSet) {
+        wait.until(ExpectedConditions.visibilityOf(element));
+        element.sendKeys(textToSet);
+    }
+    public static void WaitAndSetText (WebElement element, final String textToSet) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)